README.md

```diff
@@ -1,191 +1,249 @@
-
-## Trying PyPanther
-
-### Supported Content Types
-
-`pypanther` is under active development (available in closed beta for select customers) and currently supports the following analysis types:
-
-| Analysis Type       | Supported           |
-|---------------------|---------------------|
-| Streaming Rules     | :white_check_mark:  |
-| Data Models         | :white_check_mark:  |
-| Helper Functions    | :white_check_mark:  |
-| Built-in Content    | :white_check_mark:  |
-| Scheduled Rules     | :construction:      |
-| Lookups/Enrichments | :construction:      |
-| Saved Queries       | :construction:      |
-| Policies            | :construction:      |
-| Correlation Rules   | :construction:      |
-
-*Note: `packs` have been replaced by the `main.py` and the `get_panther_rules` function.*
-
-As more analysis types are supported, you can declare and upload using `pypanther` with the following guidance:
-1. Make sure you are on the latest `pypanther-starter-kit` and `pypanther` library by running `make update`
-2. Customize your `main.py` to include panther-managed rules or custom rules. When testing, we recommend starting with only 3-5 rules.
-3. Use `pypanther upload` to validate alerts are firing and other content is as you expect it.
-
-`pypanther` content mirrors `panther-analysis` with the addition of a `-prototype` suffix. You can use this to distinguish between v1 and v2 rules. Once `pypanther` is generally available, we will publish our migration tool and guidance to make the cutover as smooth as possible!
+| Command | Description | Example |
+|---------|-------------|---------|
+| `version` | Display CLI version | `pypanther version` |
+| `list` | List managed content | `pypanther list rules --log-types AWS.CloudTrail` |
+| `get` | Retrieve rule source | `pypanther get rule <id>` |
+| `test` | Run rule tests | `pypanther test --tags Exfiltration` |
+| `upload` | Upload to Panther | `pypanther upload --verbose` |
 
-### Required File Structure
+Use `pypanther <command> --help` for detailed usage.
 
-`pypanther`'s primary configuration file `main.py` must be located in the root directory, and all content is organized into several key directories under the `content/` folder:
+## 📝 CLI Examples
 
-- **`main.py`**: This is the main file of the repository. It controls the entire configuration for `pypanther`. The `main.py` file orchestrates which rules are imported and overridden with custom configurations.
-
-- **`content/rules/`**: This directory contains customer-defined rules that are grouped by log type family. Each folder may also include longer sample events for `RuleTest`s.
-
-- **`content/helpers/`**: The `helpers/` directory is home to generic helper functions. These functions are designed to be reusable and can be utilized either in rules or filters. Their purpose is to simplify the code in the main logic by abstracting common tasks into functions.
+### Listing Rules
```

Comment:
Should specify that these commands are going to show the *local* rules and not what's running in their Panther instance? Few folks have been confused by this in the past.

Reply:
That's a good call... I'll update it.